core/templates/search/search_advanced.html

```diff
@@ -74,16 +74,6 @@
                     <div class="col-md-10">
                         {{ form.county }}
                     </div>
-
-                    {% block search_state_field %}
-                    <div class="col-md-2">
-                        <label for="id_state">{{ form.state.label }}</label>
-                    </div>
-                    <div class="col-md-10">
-                        {{ form.state }}
```

Comment:
You could remove form.state from the `forms.py` file, iirc, in order to further wipe it out if that's what you're into